launch/CHANGELOG.rst

```diff
@@ -2,6 +2,17 @@
 Changelog for package launch
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Forthcoming
+-----------
+* Bumped patch version
+* Validate unparsed attributes and subentities in launch_xml and launch_yaml (`#468 <https://github.com/ros2/launch/issues/468>`_) (`#567 <https://github.com/ros2/launch/issues/567>`_)
+  (cherry picked from commit 00f7ccf2b38c0a05d3e81df3e0fdabb5a4039752)
+  Co-authored-by: Ivan Santiago Paunovic <ivanpauno@ekumenlabs.com>
+* Evaluate math symbols and functions in python expression (`#557 <https://github.com/ros2/launch/issues/557>`_) (`#562 <https://github.com/ros2/launch/issues/562>`_)
+  * Evaluate math symbols and functions in python expression
+  Co-authored-by: Jacob Perron <jacob@openrobotics.org>
+* Contributors: Aditya, Immanuel Martini, mergify[bot]
```

Comment:
Ideally, we should cleanup the changelogs after generating them.

```suggestion
* Validate unparsed attributes and subentities in launch_xml and launch_yaml. (`#468 <https://github.com/ros2/launch/issues/468>`_) (`#567 <https://github.com/ros2/launch/issues/567>`_)
* Evaluate math symbols and functions in python expression. (`#557 <https://github.com/ros2/launch/issues/557>`_) (`#562 <https://github.com/ros2/launch/issues/562>`_)
* Contributors: Aditya, Immanuel Martini
```

e.g.: Add dots at end of sentences, remove the commit message details, make sure the brief description of the commit is clear (if not check the PR and rewrite it), remove unnecessary items from the changelog (e.g.: bumped patch version)

Reply:
could you do this for the other changelogs and squash the commit with the existing one https://github.com/ros2/launch/pull/581/commits/06780745b374b351be4f41dad7e61db1ef2919ff?

Reply:
Done !